preservedata.js

```diff
@@ -0,0 +1,38 @@
+const names = document.getElementById('name');
+const email = document.getElementById('email');
+const text = document.getElementById('text');
+const form = document.querySelector('form');
+const body = document.querySelector('body');
+
+const retrieveData = localStorage.getItem('user');
+
+body.onload = () => {
+  if (retrieveData) {
+    const store = JSON.parse(retrieveData);
+    names.value = store.names;
+    email.value = store.email;
+    text.value = store.text;
+  }
+};
```

Comment:
- [ ] Kindly fix the object properties names as the name input is returning undefined instead of the user names
![Screen Shot 2022-09-16 at 15 02 50](https://user-images.githubusercontent.com/28307960/190645420-768af23b-3083-4359-9a23-bab4d9581b57.png)